src/util/apiUtils.js

```diff
@@ -295,17 +303,21 @@ export function parseRequestQueries(request) {
  */
 export function parseRequest(request) {
 	const baseUrl = request.server.app.API_SERVER_ROOT_URL;
-	return {
-		externalRequestOpts: {
-			baseUrl,
-			method: request.method,
-			headers: parseRequestHeaders(request),  // make a copy to be immutable
-			mode: 'no-cors',
-			time: true,  // time the request for logging
-			agentOptions: {
-				rejectUnauthorized: baseUrl.indexOf('.dev') === -1
-			},
+	const externalRequestOpts = {
+		baseUrl,
+		method: request.method,
+		headers: parseRequestHeaders(request),  // make a copy to be immutable
+		mode: 'no-cors',
+		time: true,  // time the request for logging
+		agentOptions: {
+			rejectUnauthorized: baseUrl.indexOf('.dev') === -1
 		},
+	};
+	if (request.mime === 'multipart/form-data') {
+		externalRequestOpts.formData = request.payload;
```

Comment:
Where does `request.payload` come from? That's not the same as the dispatch action payload right?